### PR TITLE
RT-DETR r50 demo

### LINKED ISSUE #558: RT-DETR R50 Demo
Just a simple demo, showing the detector and the default draw func.

### PR DESCRIPTION
**Pull request by abramov-oleg**

### CODE REVIEW

File: samples/rtdetr/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# RT-DETR R50 Demo
+
+The sample shows how RT-DETR model can be used in a Savant module.
+
+The detector model was prepared in the ONNX format using instructions from [DeepStream-Yolo repo](https://github.com/marcoslucianops/DeepStream-Yolo/blob/master/docs/RTDETR.md).
+
+Weights used: `v0.1/rtdetr_r50vd_6x_coco_from_paddle.pth`  from the [RT-DETR releases](https://github.com/lyuwenyu/storage/releases).
+
+Run the demo:
```
Comment (denisvmedyantsev) [inline samples/rtdetr/README.md]: Please refactor according to other samples: add prerequisite and the build engine stage

File: samples/rtdetr/docker-compose.x86.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+      - ../../models/rtdetr:/models
+      - ../../downloads/rtdetr:/downloads
+      - .:/opt/savant/samples/rtdetr
+    command: samples/rtdetr/module.yml
+    environment:
+      - ZMQ_SRC_ENDPOINT=router+bind:ipc:///tmp/zmq-sockets/input-video.ipc
+      - ZMQ_SINK_ENDPOINT=pub+bind:ipc:///tmp/zmq-sockets/output-video.ipc
+      - FPS_PERIOD=1000
```
Comment (denisvmedyantsev) [inline samples/rtdetr/docker-compose.x86.yml]: Refactor to use the new FPS meter. Or just remove FPS_PERIOD, I don't think it's important to change the FPS measurement period here.

**Review by denisvmedyantsev — CHANGES_REQUESTED** [review]

File: samples/rtdetr/README.md (hunk 1 above)
Comment (abramov-oleg) [reply samples/rtdetr/README.md]: Refactored README.

File: samples/rtdetr/docker-compose.x86.yml (hunk 2 above)
Comment (abramov-oleg) [reply samples/rtdetr/docker-compose.x86.yml]: Removed FPS_PERIOD.

**Review by denisvmedyantsev — APPROVED** [review]

**Pull request merged**